prime-dotnet-webapi/Models/PlrProvider.cs

```diff
@@ -97,5 +97,59 @@ public class PlrProvider : BaseAuditable
         public string Email { get; set; }
 
         public string ConditionCode { get; set; }
+
+
+        /// <summary>
+        /// All properties should be update-able except for the keys:  Id and Ipc
+        /// </summary>
+        /// <param name="newer"></param>
+        public void Update(PlrProvider newer)
```

Comment:
Is there a reason this is all manually assigned? AutoMapper or context -> Entity -> SetValues() would work here, I would think. See this snippet from UpdateEnrollee():
```C#
_context.Entry(enrollee).CurrentValues.SetValues(updateModel);
```